src/.vuepress/markdown/link.js

```diff
@@ -0,0 +1,95 @@
+// markdown-it plugin for:
+// 1. adding target="_blank" to external links
+// 2. converting internal links to <router-link>
+
+const url = require('url')
+
+const indexRE = /(^|.*\/)(index|readme).md(#?.*)$/i
+
+module.exports = (configuration) => {
+  return (md) => {
+    let hasOpenRouterLink = false
+    let hasOpenExternalLink = false
+
+    md.renderer.rules.link_open = (tokens, idx, options, env, self) => {
+    const { base } = configuration;
```

Comment:
* indentation mishap
* inconsistent use of `;`

We should probably add a linter check to that directory.

Reply:
The use of `;` is still inconsistent with the rest of the files.

Reply:
Fixed.